Make.inc

```diff
@@ -488,7 +488,7 @@ $(error Sanitizers are only supported with clang. Try setting SANITIZE=0)
 endif
 CC := $(CROSS_COMPILE)gcc
 CXX := $(CROSS_COMPILE)g++
-JCFLAGS := -std=gnu99 -pipe $(fPIC) -fno-strict-aliasing -D_FILE_OFFSET_BITS=64
+JCFLAGS := -std=gnu99 -fexcess-precision=standard -pipe $(fPIC) -fno-strict-aliasing -D_FILE_OFFSET_BITS=64
```

Comment:
Adding `-mfpmath=sse` where we set `-march` (in Make.inc) might be better?

Reply:
I don't think that does what we want. The problem is we need to tell gcc not to use 80 bit operations when we want 64 bit.

Reply:
That is what that does

Reply:
what's the difference between `-fexcess-precision=standard` and `-mfpmath=sse `?

Reply:
quoth the docs `-fexcess-precision=standard is not implemented for languages other than C. On the x86, it has no effect if -mfpmath=sse or -mfpmath=sse+387 is specified; in the former case, IEEE semantics apply without excess precision, and in the latter, rounding is unpredictable.`

Reply:
argh. `-mfpmath=sse` only works on x86. Are we worried about hitting this bug in the future on arm?

Reply:
No, arm doesn't have an 80bit fp unit.